src/components/fields/ImageField.vue

```diff
@@ -7,7 +7,7 @@
         </div>
       </template>
       <template v-else>
-        <div class="btn btn-default has-image" :style="'background-image: url(' + valueToShow + ')'" @click.prevent="openFilePicker"></div>
+        <div class="btn btn-default has-image" :style="'background-image: url(' + valueToShow.url + ')'" @click.prevent="openFilePicker"></div>
```

Comment:
you might want to account for existing settings, e.g. `valueToShow.url || valueToShow`